### DISCUSSION TITLE
gatsby-plugin-manifest: Specify iOS Icon

### DISCUSSION THREAD

**Discussion by agiron123** [gatsbyjs/gatsby#28148] [Help]
My team wishes to use a different favicon for iOS users. Is it possible to do this in gatsby-plugin-manifest?

I see the code in the plugin's gatsby-ssr which generates the link tags to go into the head of the page and it seems that it uses the same icon as the other favicons being generated:
https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-plugin-manifest/src/gatsby-ssr.js

Are my assumptions here correct that it's not possible to specify an icon to use for iOS only, or is there another configuration option I can use?

**Comment by LekoArts** [comment]
You can define all icons individually in manual mode: https://github.com/gatsbyjs/gatsby/tree/master/packages/gatsby-plugin-manifest#manual-mode-configuration
